python/paddle/fluid/layers/nn.py

```diff
@@ -9893,3 +9894,73 @@ def huber_loss(input, label, delta):
                  'Residual': residual},
         attrs={'delta': delta})
     return out
+
+
+@templatedoc()
+def tree_conv(nodes_vector,
+              edge_set,
+              output_size,
+              num_filters=1,
+              max_depth=2,
+              act='tanh',
+              param_attr=None,
+              bias_attr=None,
+              name=None):
+    """ 
+    ${comment}
+    		
+    Args:
+        nodes_vector(${nodes_vector_type}): ${nodes_vector_comment}
+        edge_set(${edge_set_type}): ${edge_set_comment}
+        output_size(int): output feature width
+        num_filters(int): number of filters, Default 1
+        max_depth(int): max depth of filters, Default 2
+        act(str): activation function, Default tanh
+        param_attr(ParamAttr): the parameter attribute for the filters, Default None
+        bias_attr(ParamAttr): the parameter attribute for the bias of this layer, Default None
+        name(str): a name of this layer(optional). If set None, the layer will be named automatically, Default None
+
+    Returns:
+        out(${out_type}): ${out_comment}
+
+    Examples:
+        .. code-block:: python
+
+          nodes_vector = layers.data(name='vectors', shape=[None, 10, 5], dtype='float32)
+          # None for batch size, 10 for max_node_size of dataset, 5 for vector width
+          edge_set = layers.data(name='edge_set', shape=[None, 10, 2], dtype='float32')
```

Comment:
Is line9931 correct? I change it to `edge_set = layers.data(name='edge_set', shape=[32, 10, 2], dtype='float32')`, and log is
```
$ python
Python 2.7.3 (default, Sep 26 2018, 13:48:39)
[GCC 4.4.6 20120305 (Red Hat 4.4.6-4)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import paddle.fluid as fluid
>>> nodes_vector = fluid.layers.data(name='vectors', shape=[32, 10, 5], dtype='float32')
>>> edge_set = fluid.layers.data(name='edge_set', shape=[32, 10, 2], dtype='float32')
>>> out_vector = fluid.layers.tree_conv(nodes_vector, edge_set, 6, 1, 2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/luotao/.jumbo/lib/python2.7/site-packages/paddle/fluid/layers/nn.py", line 11053, in tree_conv
    attrs={'max_depth': max_depth})
  File "/home/luotao/.jumbo/lib/python2.7/site-packages/paddle/fluid/layer_helper.py", line 43, in append_op
    return self.main_program.current_block().append_op(*args, **kwargs)
  File "/home/luotao/.jumbo/lib/python2.7/site-packages/paddle/fluid/framework.py", line 1654, in append_op
    attrs=kwargs.get("attrs", None))
  File "/home/luotao/.jumbo/lib/python2.7/site-packages/paddle/fluid/framework.py", line 1058, in __init__
    self.desc.infer_shape(self.block.desc)
paddle.fluid.core.EnforceNotMet: Enforce failed. Expected edge_dims[2] == 2, but received edge_dims[2]:10 != 2:2.
Input(EdgeSet) dim[2] should be 2 at [/home/luotao/Paddle/paddle/fluid/operators/tree_conv_op.cc:72]
PaddlePaddle Call Stacks:
```